composer.json

```
@@ -10,23 +10,21 @@
     ],
     "homepage": "https://github.com/kyslik/laravel-filterable",
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Martin Kiesel",
-            "email": "martin.kiesel@gmail.com",
-            "role": "Developer"
-        }
-    ],
+    "authors": [{
+        "name": "Martin Kiesel",
+        "email": "martin.kiesel@gmail.com",
+        "role": "Developer"
+    }],
     "require": {
         "php": ">=7.1",
-        "illuminate/support": "^7.0",
-        "illuminate/database": "^7.0",
-        "illuminate/http": "^7.0"
+        "illuminate/support": "^7.0|^8.0",
+        "illuminate/database": "^7.0|^8.0",
+        "illuminate/http": "^7.0|^8.0"
     },
     "require-dev": {
-        "laravel/framework": "^7.0",
-        "orchestra/testbench": "^5.0",
-        "phpunit/phpunit": "^8.0"
+        "laravel/framework": "^7.0|^8.0",
+        "orchestra/testbench": "^5.0|^6.0",
+        "phpunit/phpunit": "^8.0|^9.0"
```

```suggestion
        "laravel/framework": ">=7.0",
        "orchestra/testbench": ">=5.0",
        "phpunit/phpunit": ">=8.0"
```

Let's see if this works 🧐 .